schemas/choice.json

```diff
@@ -18,7 +18,12 @@
       "description": "Description of the essence of the choice in this node.",
       "type": "string",
       "minLength": 1
+    },
+    "acceptingStatus": {
```

Comment:
let's not mix:
* model for saving the progress and
* model for describing the story

this boolean flag is only will be needed to store progress for one specific customer, let's not do it here

Reply:
`+`